gulpfile.js

```diff
@@ -9,6 +8,9 @@ var uglify = composer(uglifyes, console);
 
 // Check the NODE_ENV environment variable
 var isDev = process.env.NODE_ENV === 'development';
+// Allow overriding of jellyfin-web directory
+var WEB_DIR = process.env.JELLYFIN_WEB_DIR || 'node_modules/jellyfin-web/dist';
+console.info('Using jellyfin-web from', WEB_DIR);
```

Comment:
needs a space

Reply:
A space should be added automatically since in this form we're logging multiple values a space is inserted in between them. My console output is below.

```
Using jellyfin-web from node_modules/jellyfin-web/dist
```